### ISSUE TITLE
Transferring pokemon set to not transfer 

### ISSUE THREAD

**Issue by JewBagel** [finndev/PokeBuddy#832]
as the title says as im trying to evolve a pokemon that is on both do not transfer and the evolve list it will still transfer them..

**Comment by OnlyMeFelix** [comment]
Nope, so long as you have the evolved form on the do not transfer list. I made this mistake before. IE told it to evolve Bulbasaur, but then transferred the Ivysaur as i didn't have this checked.
